src/frontend/components/dashboard/MapTab/mapboxStyle.js

```diff
@@ -18,7 +18,9 @@ const mbStyle = {
     },
     composite: {
       type: 'vector',
-      url: 'mapbox://jeremiak.4hjgevzj', //'mapbox://jeremiak.27hyzjbx',
+      tiles: [
+        'https://storage.googleapis.com/maptiles-mlab-sandbox/piecewise/{z}/{x}/{y}.pbf',
```

Comment:
For a future issue, we should plan for the storage URL in GCS to accommodate multiple releases of tile sets, following the FCC releases. So something like:
* `https://storage.googleapis.com/maptiles-mlab-sandbox/piecewise/2020Q12/{z}/{x}/{y}.pbf`
* `https://storage.googleapis.com/maptiles-mlab-sandbox/piecewise/2020Q34/{z}/{x}/{y}.pbf`
* `https://storage.googleapis.com/maptiles-mlab-sandbox/piecewise/2019Q12/{z}/{x}/{y}.pbf`
...etc